example.py

```diff
@@ -422,10 +425,12 @@ def test_upload_and_download_a_file():
                 assert "foobarbaz" == data
 
 
-def test_upload_and_download_a_file_using_sftp():
-    "same as `test_upload_and_download_a_file`, but using SFTP to transfer files"
-    with settings(transfer_protocol="sftp"):
-        return test_upload_and_download_a_file()
+def test_upload_and_download_a_file_coverage_bump():
+    """tests uploading and downloading a file using all three transfer protocols.
+    this is covered more thoroughly in the `./project-tests.sh` script, this is just
```

Comment:
"this is covered more thoroughly in the `./project-tests.sh` script and is just to bump test coverage."